tests/plugins/test_orchestrate_build.py

```diff
@@ -136,7 +137,7 @@ def mock_create_worker_build(**kwargs):
 
     (flexmock(OSBS)
         .should_receive('get_build_logs')
-        .and_yield('line-%d' % line for line in range(10)))
+        .and_yield('Lińe-%d' % line for line in range(10)))
```

Comment:
I don't think this would trigger the scenario we want.
Maybe we need to mock at the API level? But that seems something this plugin shouldn't be concerned with.